matmul.hip

```diff
@@ -198,54 +211,81 @@ void checkMmtResults(Type A_type, Type B_type, Type C_type,
   abort();
 }
 
-typedef void (*mmt_func_t)(const void *, const void *, void *, int, int);
+typedef void (*mmt_func_t)(const void *, const void *, void *, void *, int, int,
+                           int);
```

Comment:
nit: `using` are usually more readable than `typedef`s.

Reply:
oh, that's preexisting. I remember trying `using` because I agree with you, and failing to find how to port the old C function pointer typedef syntax to that. Feel free :-)